### ISSUE TITLE
Why is there no test suite included?

### ISSUE THREAD

**Issue by kristianmandrup** [GoogleChrome/guitar-tuner#12]
Just wondering...

**Comment by paullewis** [comment]
You're right to hold me to a higher standard. As it happens I find this kind of stuff difficult to unit test: the audio processing doesn't really work from a CLI, etc. And generally speaking I find that UI testing is patchy at best. Ideas on the approach would be welcome!

**Comment by paullewis** [comment]
(Also sorry, I only just noticed the issue!)
